packages/ove-ui-launcher/src/steps/StateConfiguration.jsx

```diff
@@ -68,18 +68,18 @@ export default class StateConfiguration extends Component {
         return {
             stateVal: (data.mode === Constants.Mode.NEW || data.state !== ''),
             urlVal: (data.mode !== Constants.Mode.NEW ||
-                (Constants.VALID_URL_REGEX.test(data.url) && this._validateYoutubeURL(data)))
+                (Constants.VALID_URL_REGEX.test(data.url) && this._validateVideoURL(data)))
         };
     }
 
-    _validateYoutubeURL(data) {
-        return this.state.app !== Constants.App.VIDEOS || data.mode !== Constants.Mode.NEW || Constants.YOUTUBE_URL_REGEX.test(data.url);
+    _validateVideoURL(data) {
+        return this.state.app !== Constants.App.VIDEOS || data.mode !== Constants.Mode.NEW || Constants.YOUTUBE_URL_REGEX.test(data.url) || !data.url.includes("youtube.com");
```

Comment:
If it is not a YouTube URL, then we need to run a regular URL validation instead of skipping the validation. Can we fix that?

Also, youtube has variants that do not include ".com", but we only accept the variant that includes ".com", so we will need to test the existence of youtube in the URL, instead of youtube.com. This is also how the Videos App works.

Reply:
@jamesscottbrown, please also revert this fix that used double quotes instead of single quotes. Codacy was not properly configured previously due to the missing ESLint configuration, which was reporting a number of incorrect errors. This has now been fixed.